LibreNMS/OS/Endrun.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * EndRun.php
+ *
+ * EndRun Tempus LX NTP devices
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @package    LibreNMS
+ * @link       http://librenms.org
+ * @copyright  2020 Hans Erasmus
+ * @author     Hans Erasmus <erasmushans27@gmail.com>
+ */
+
+namespace LibreNMS\OS;
+
+use LibreNMS\Interfaces\Discovery\OSDiscovery;
+use LibreNMS\OS;
+
+class Endrun extends OS implements OSDiscovery
+{
+    public function discoverOS(): void
+    {
+        $device = $this->getDeviceModel();
+        $info = snmp_getnext_multi($this->getDevice(), 'cntpVersion cdmaVersion', '-OQUs', 'TEMPUSLXUNISON-MIB');
+        $info2 = snmp_getnext_multi($this->getDevice(), 'snmpSetSerialNo', '-OQUs', 'SNMPv2-MIB');
```

Comment:
why a second get next call? also, plain snmpgets are better if you know the index will be just 0

Reply:
Sorry for being daft. I used it in the way I did, because of the different MIBS needed to get all the values.

Reply:
not daft, just inexperienced.  No worries.